src/stories/Library/shadow/Shadow.tsx

```diff
@@ -0,0 +1,25 @@
+const shadowValues = [
+  "shadow-low",
+  "shadow-medium",
+  "shadow-high",
+  "shadow-low-hover",
+  "shadow-medium-hover",
+  "shadow-high-hover",
+];
+
+export const Shadow = () => {
+  return (
+    <div className="shadow-box-container m-24">
+      <p className="text-body-large">Shadows</p>
+      {shadowValues.map((shadowValue) => (
+        <div className={shadowValue}>
```

Comment:
mmm, I think this will be throwing a React "each child must have a unique key prop" error in the console. Can you please check and potentially add a key={generateId(index)} to the div at line 15?

Reply:
You are right. I have added a key to the map.